test/integration/shared/host_finders.rb

```diff
@@ -1,30 +1,41 @@
 module HostFinders
   extend ActiveSupport::Concern
 
+  def disable_interface_modal_animation
+    page.evaluate_script('document.getElementById("interfaceModal").classList.remove("fade")')
```

Comment:
removes "fade" from modal, what makes it appear instantly and prevent the movement of the modal on the page, where the driver had trouble to count the click coordinates properly.